package.json

```diff
@@ -677,6 +689,17 @@
                         "default": true,
                         "scope": "application",
                         "description": "Add difficulty badge and colorize problems files in explorer tree."
+                    },
+                    "leetcode.problems.sort-strategy": {
```

Comment:
nit: sort-strategy -> sortStrategy

And please add it to the setting section in readme